cloudwash/providers/ec2.py

```diff
@@ -4,8 +4,32 @@
 from cloudwash.utils import dry_data
 from cloudwash.utils import echo_dry
 from cloudwash.utils import total_running_time
+from cloudwash.utils import discovery_timestamp
 from cloudwash.client import compute_client
+import wget
+import tarfile
+import subprocess
+from os.path import exists
+import os
 
+def delete_ocp(ocp_name):
+    region = settings.providers.ec2.region
+    logger.info("Delete OCP '" + ocp_name + "'")
+    f = open("metadata.json", "w")
+    f.write("{\"clusterName\":\"" + ocp_name + "\",\"infraID\":\"" + ocp_name + "\",\"aws\":{\"region\":\"" + region + "\",\"identifier\":[{\"kubernetes.io/cluster/" + ocp_name + "\":\"owned\"}]}}")
```

Comment:
I think you can create a plain dictionary and then use `json` python library to dump it to a json file.